src/cudamatrix/cu-matrix.h

```diff
@@ -250,6 +250,10 @@ class CuMatrixBase {
   /// Copy vector into specific column of matrix.
   void CopyColFromVec(const CuVectorBase<Real> &v, const MatrixIndexT col);
 
+  /// Compute the rectified linear unit function; element by element,
+  /// *this = ParametricRelu(src).
```

Comment:
You need to document the actual equations here.  Parametric ReLU is pretty obscure and not something one can assume knowledge of.